src/OpenTK/Platform/Linux/LinuxDisplayDriver.cs

```diff
@@ -92,12 +78,15 @@ public LinuxDisplay(int fd, IntPtr c, IntPtr e, IntPtr r)
         }
     }
 
-    internal class LinuxDisplayDriver : DisplayDeviceBase
+    internal class LinuxDisplayDriver : IDisplayDeviceDriver
     {
         private readonly int FD;
 
         private readonly Dictionary<int, int> DisplayIds =
             new Dictionary<int, int>();
+        
+        private DisplayDevice _primary;
```

Comment:
It's typical that private fields are prefixed with an underscore. However, we haven't set a standard for the codebase yet.